src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -72,29 +74,52 @@ class ComplianceCard extends Component {
-                                        </StackItem>
+                                    getTopThreePolicies(complianceSummary).map((element, index) =>
+                                        <div className="ins-c-compliance__row" key={ index }>
+                                            <div className="ins-c-compliance__row-item">
+                                                <PieChart
+                                                    containerWidth={ 75 }
+                                                    containerHeight={ 75 }
+                                                    ariaDesc="Operating systems used"
+                                                    ariaTitle="Pie chart operating systems"
+                                                    constrainToVisibleArea={ true }
+                                                    data={ [
+                                                        { x: element.attributes.name, y: element.attributes.score * 100 },
+                                                        { x: 'empty', y: 100 }
+                                                    ] }
+                                                    height={ 100 }
+                                                    labels={ ({ datum }) => `${datum.x}: ${datum.y}` }
+                                                    padding={ pieChartPadding }
+                                                    width={ 100 }
+                                                    colorScale={ ['#002f5d', '#06c', '#8bc1f7'] }
+                                                />
+                                            </div>
+                                            <div className="ins-c-compliance__row-item">
+                                                <Stack>
+                                                    <StackItem>
+                                                        <Button
+                                                            className="ins-c-compliance__policy-link"
+                                                            component="a"
+                                                            href={ `/${UI_BASE}/compliance/policies/` }
+                                                            variant="link"
+                                                            isInline
+                                                        >
+                                                            {element.attributes.name}
+                                                        </Button>
+                                                    </StackItem>
+                                                    <StackItem>
+                                                        <Split gutter='sm'>
+                                                            <SplitItem>
+                                                                {element.attributes.compliant_host_count} systems
```

Comment:
Need to build this string out in messages.js (and then compile em), gotta keep up with translations 🦹‍♀️

Reply:
BESIDES this is a prime place to leverage the power pluralization of translations, otherwise this runs the risk of reading `1 systems` check it out in messages.js